### PR TITLE
chore(kms): Add samples from ruby-docs-samples

### PR DESCRIPTION
**Pull request by quartzmo**
refs: GoogleCloudPlatform/ruby-docs-samples#589

### CODE REVIEW

**Comment by quartzmo** [comment]
The Samples - Linux CI errors are all of cause: `Google::Cloud::ResourceExhaustedError: 8:Quota exceeded for quota metric 'Write requests' and limit 'Write requests per minute' of service 'cloudkms.googleapis.com' for consumer...`

The RPC method is either `create_crypto_key ` or other `crypto_key ` CRUD.

**Comment by quartzmo** [comment]
I will update the tests to use a persistent key ring and a persistent set of keys.

File: google-cloud-kms/samples/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+    ```text
+    $ bundle exec ruby snippets.rb
+    ```
+
+    The output will show the help text.
+
+## Run the tests
+
+Run the tests for the samples by running `bundle exec rake test`.
```
Comment (dazuma) [inline google-cloud-kms/samples/README.md:51]: Can you quickly document how to build the test fixtures here?

Comment (quartzmo) [reply google-cloud-kms/samples/README.md:51]: Yes.

File: google-cloud-kms/samples/README.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+Run the acceptance tests for the samples by running `bundle exec rake test`.
+
+## Contributing changes
+
+* See [CONTRIBUTING.md](../CONTRIBUTING.md)
+
+## Licensing
+
+* See [LICENSE](../LICENSE)
```
Comment (dazuma) [inline google-cloud-kms/samples/README.md:63]: Should be `LICENSE.md`.

Comment (quartzmo) [reply google-cloud-kms/samples/README.md]: Removed these links, since other samples READMEs do not contain them.

**Pull request merged**